bgpd/bgp_evpn.c

```diff
@@ -2994,23 +3024,27 @@ static int process_type5_route(struct peer *peer, afi_t afi, safi_t safi,
 		pfx += 16;
 	}
 
-	label_pnt = (mpls_label_t *)pfx;
+	/* Get the VNI (in MPLS label field). Stored as bytes here. */
+	memset(&label, 0, sizeof(label));
+	memcpy(&label, pfx, BGP_LABEL_BYTES);
+	pfx += 3;
 
```

Comment:
replace 3 with BGP_LABLE_BYTEs

Reply:
Removed this as SA was throwing an error here as we dont use the pfx value later.